server/jetstream_api.go

```diff
@@ -509,10 +503,14 @@ func (s *Server) jsTemplateCreateRequest(sub *subscription, c *client, subject,
 	if err != nil {
 		resp.Error = jsError(err)
 		s.sendAPIResponse(c, subject, reply, string(msg), s.jsonResponse(&resp))
+		return
 	}
 	t.mu.Lock()
 	tcfg := t.StreamTemplateConfig.deepCopy()
 	streams := t.streams
+	if streams == nil {
```

Comment:
we can also fix this in LookupStreamTemplate and where its being created, but I didnt want to go in there and fiddle things, we can if you prefer.